deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,347 @@
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/cluster"
+                  }
+                },
+                "name": "odhargo-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/odhargo"
+                  }
+                },
+                "name": "odhargo"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/master"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v0.9.0"
+              }
+            ],
+            "version": "v0.9-branch-openshift"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: AI/Machine Learning, Big Data
+    description: "Open Data Hub Operator for deployment and management of Open Data Hub and Kubeflow on OpenShift"
+    support: Open Data Hub
+    repository: https://github.com/opendatahub-io/opendatahub-operator
+    createdAt: '2020-11-09T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v0.9.0
+    certified: 'False'
+  name: opendatahub-operator.v0.9.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: Open Data Hub
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide as complete a solution as possible for both hobby and enterprise users.
+
+    ### Components
+    * Open Data Hub Dashboard - ***NEW*** A webui dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support. Notebook with support Elyra and JupyterLab
```

Comment:
Can we replace "webui" with web portal? or just web dashboard?

Reply:
Updated to say `web dashboard`